src/main/java/teammates/common/util/Templates.java

```diff
@@ -411,21 +411,30 @@ public static String populateTemplate(String template, String... keyValuePairs)
             public static final String RANK_AVERAGE = "${averageRank}";
             public static final String RANK_EXCLUDING_SELF_AVERAGE = "${averageRankExcludingSelf}";
             public static final String RANK_EDIT_FORM_OPTION_FRAGMENTS = "${rankEditFormOptionFragments}";
-            public static final String RANK_MIN_OPTIONS_TO_BE_RANKED = "${minOptionsToBeRanked}";
-            public static final String RANK_MAX_OPTIONS_TO_BE_RANKED = "${maxOptionsToBeRanked}";
             public static final String RANK_ARE_DUPLICATES_ALLOWED_VALUE = "${areDuplicatesAllowedValue}";
             public static final String RANK_ARE_DUPLICATES_ALLOWED_CHECKED = "${areDuplicatesAllowedChecked}";
-            public static final String RANK_DISPLAY_MIN_OPTIONS_HINT = "${displayMinOptionsHint}";
-            public static final String RANK_IS_MIN_OPTIONS_TO_BE_RANKED_ENABLED = "${isMinOptionsToBeRankedEnabled}";
-            public static final String RANK_DISPLAY_MAX_OPTIONS_HINT = "${displayMaxOptionsHint}";
-            public static final String RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED = "${isMaxOptionsToBeRankedEnabled}";
             public static final String RANK_SUBMISSION_FORM_OPTION_FRAGMENTS = "${rankSubmissionFormOptionFragments}";
             public static final String RANK_OPTION_RECIPIENT_DISPLAY_NAME = "${optionRecipientDisplayName}";
+            public static final String RANK_MIN_OPTIONS_TO_BE_RANKED = "${minOptionsToBeRanked}";
+            public static final String RANK_MAX_OPTIONS_TO_BE_RANKED = "${maxOptionsToBeRanked}";
+            public static final String RANK_IS_MIN_OPTIONS_TO_BE_RANKED_ENABLED = "${isMinOptionsToBeRankedEnabled}";
+            public static final String RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED = "${isMaxOptionsToBeRankedEnabled}";
+            public static final String RANK_DISPLAY_MIN_OPTIONS_HINT = "${displayMinOptionsHint}";
+            public static final String RANK_DISPLAY_MAX_OPTIONS_HINT = "${displayMaxOptionsHint}";
+            public static final String RANK_MIN_RECIPIENTS_TO_BE_RANKED = "${minRecipientsToBeRanked}";
+            public static final String RANK_MAX_RECIPIENTS_TO_BE_RANKED = "${maxRecipientsToBeRanked}";
+            public static final String RANK_DISPLAY_MIN_RECIPIENTS_HINT = "${displayMinRecipientsHint}";
+            public static final String RANK_DISPLAY_MAX_RECIPIENTS_HINT = "${displayMaxRecipientsHint}";
+            public static final String RANK_IS_MIN_RECIPIENTS_TO_BE_RANKED_ENABLED = "${isMinRecipientsToBeRankedEnabled}";
+            public static final String RANK_IS_MAX_RECIPIENTS_TO_BE_RANKED_ENABLED = "${isMaxRecipientsToBeRankedEnabled}";
```

Comment:
Let's make the order uniform: `min-*s`, `max-*s`, `is-min-*s`, `is-max-*s`, `display-min-*s`, `display-max-*s`. You can tweak the order, but the idea is to make it similar in both options and recipients.